lesson7/thingspeak_cpu_loop.py

```diff
@@ -2,12 +2,19 @@
 from time import localtime, strftime
 import psutil
 import time
+from pickle import dump as pickle_dump
+from pickle import load as pickle_load
+from os.path import exists as file_exists
+from os.path import isfile as is_file
+PICKLE_FILE_NAME = "API_KEY.pickle"
+API_KEY_INDEX = "API_KEY"
```

Comment:
Key for storage in a dictionary. Although this is unnecessary right now (given we are storing exactly 1 item), this could be expanded in the future if other information needs to be similarly stored.